packages/web-monetization-wext/src/content/DocumentMonetization.ts

```diff
@@ -84,9 +84,15 @@ export class DocumentMonetization {
           this.window.location.origin
         )
       }
-      if (this.request && (state === 'stopped' || state === 'pending')) {
```

Comment:
Should these changes be applied to oauth-scripts as well?

Reply:
I'm not entirely sure about those exact changes, and the implications of pre vs post pay, but I believe they should both send pending/started in quick succession as per the extension